assets/images/general/collection_start_here.svg

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Is there any way we can use text for this instead? Otherwise it can't be translated.

OK to do this in a separate PR if you like, but please file an issue if that's the case!